lib/import/brca/providers/oxford/oxford_handler.rb

```diff
@@ -23,6 +25,29 @@ def process_fields(record)
             res&.each { |cur_genotype| @persister.integrate_and_store(cur_genotype) }
           end
 
+          def brca_file?
+            file_name = @batch.original_filename
+            file_path = file_name.slice(0, file_name.rindex('/'))
```

Comment:
Could you use `File.dirname(@batch.original_filename)` here?

Reply:
file_name variable (line 28) is still required for line 31. So do you want me to just replace line 30 with file_path=File.dirname(file_name)?

Reply:
You could still assign `file_name = @batch.original_filename` and then do `File.dirname(file_name)`. It's the `file_name.slice(0, file_name.rindex('/'))` that I don't think we need, the `File` class has useful methods for dealing with file names/paths/directories.

Reply:
@ollietulloch  Changes made and tested- see commit [d4d9719](https://github.com/NHSDigital/data_management_system/pull/56/commits/d4d97193ceab1a562b4be9cd728554a5b8e3526b)